policies/design-process.md

```diff
@@ -0,0 +1,154 @@
+Policy for Creating Designs
+===========================
+
+From time to time the OTC may require that certain problems should have a
+design document created for them before implementation should start. In that
+case this policy will apply.
```

Comment:
It's unclear in which case the policy needs to applied. Will this be required for all new features? Or when we think the feature reaches some threshold of size, difficulty, ... ? Should 1 OTC member just ask for it? What if there is a disagreement that one is needed or not? Can non OTC members ask for it?

Reply:
This only applies if an OTC *decision* occurs. Its not sufficient for 1 OTC member to ask for it. If consensus cannot be reached then a vote would be required. It's not for all new features and there are no formal thresholds defined - only those features where OTC collectively decides it is required. Possibly future amendments might add more rules in this area - but for now I think this is sufficient. I've clarified the first paragraph to say:

"From time to time the OTC may decide that certain problems should have a
design document created for them before implementation should start. Such a
decision should be minuted in OTC meeting minutes or recorded via a vote. In
that case this policy will apply."